hydrus/core/images/HydrusImageHandling.py

```diff
@@ -144,6 +145,16 @@ def GenerateNumPyImage( path, mime, force_pil = False ) -> numpy.array:
         
         return HydrusImageNormalisation.StripOutAnyUselessAlphaChannel( numpy_image )
         
+
+    if mime == HC.APPLICATION_KRITA:
+
+        if HG.media_load_report_mode:
+            
+            HydrusData.ShowText( 'Loading KRA' )
+
+        pil_image = HydrusKritaHandling.MergedPILImageFromKRA( path )
```

Comment:
So this will use the thumbnail if the merged image is not available? That is not desired, Hydrus can't handle displaying any image that is not the exact dimensions in the DB.